docs/cli.md

```diff
@@ -197,6 +197,56 @@ Usage:
       --validate=false: If true, use a schema to validate the input before sending it
 ```
 
+#### resize
+Set a new size for a resizable resource (currently only Replication Controllers)
+
+Resize also allows users to specify one or more _preconditions_ for the resize action.  The two currently supported options are current size (```--current-size```) or resource version (```--resource-version```).  If a precondition is specified, it is validated before the resize is attempted, and it is guaranteed that the precondition holds true when the resize is sent to the server.
+
+Examples:
+```sh
+  $ kubectl resize controllers foo 3
+  resized
+```
+
+```sh
+  # will only execute if the current size is 3
+  $ kubectl resize --current-size=2 controllers foo 3
+```
+
+Usage:
+```
+  kubectl resize [-r/--resource-version] [-s/--current-size] <resource> <id> <count> [flags]
```

Comment:
Got rid of optionals, didn't really like them anyway.

Regarding your other comment, I wrote (but I guess I didn't press comment)

I'm using the (I presume) standard [```ResourceFromArgs(...)``` function](https://github.com/GoogleCloudPlatform/kubernetes/blob/b8333bdeeffc9ecab895a3ebefe70294993c55e3/pkg/kubectl/cmd/resource.go#L127) 

Which I assume will pick up all of the write flags/patterns (and if not we should change that function...)

Because Cobra Commander uses the [POSIX flags library for golang](https://github.com/ogier/pflag) and points to the [Gnu POSIX extensions](http://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) spec states: "Options may be supplied in any order, or appear multiple times. The interpretation is left up to the particular application program."

I believe that flag ordering doesn't matter.  (I'm happy to adjust if you think that it reads better)